charts/loki-distributed/values.yaml

```diff
@@ -881,6 +881,8 @@ compactor:
 ruler:
   # -- Specifies whether the ruler should be enabled
   enabled: false
+  # -- Kind of deployment [StatefulSet/Deployment]
+  kind: StatefulSet
```

Comment:
I think we should keep this as a Deployment by default. So that the current deployments of the Ruler won't change for other users.

Reply:
Makes sense :+1: